firefox_puppeteer/ui/toolbars.py

```diff
@@ -11,55 +11,53 @@
 
 
 class NavBar(BaseLib):
-    """ The NavBar class provides access to elements contained in the
-    navigation bar as well as the locationbar.
-    """
+    """Provides access to the DOM elements contained in the
+    navigation bar as well as the location bar."""
 
     @property
     def back_button(self):
-        """ Provides access to the back button from the navbar ui.
+        """Provides access to the DOM element back button in the navbar.
 
-        :returns: The back button element.
+        :returns: Reference to the back button.
```

Comment:
Both changes are not what we have in the tabs module and for the identity class, but those things we could fix later if really necessary.